app/facades/monster_facade.rb

```diff
@@ -1,5 +1,5 @@
 class MonsterFacade
-  def monster_list
+  def self.monster_list
```

Comment:
Curious why you chose to change this to a class method

Reply:
We chose to change this to a class method so that in the query itself we would not have to initialize a facade. This allowed us to eliminate either stringing together `Facade.new.method` or instantiating on another line.